Integrations/Cisco-umbrella/Cisco-umbrella.py

```diff
@@ -1371,6 +1399,38 @@ def get_whois_for_domain_command():
             'LastObserved': res['auditUpdatedDate'],
         }
 
+        # 5.0 update - context indicators
+        admin = {
+            'Country': res.get('administrativeContactCountry', ''),
+            'Email': res.get('administrativeContactEmail', ''),
+            'Name': res.get('administrativeContactName', ''),
+            'Phone': res.get('administrativeContactTelephone', '')
+        }
+        registrant = {
+            'Country': res.get('registrantCountry', ''),
+            'Email': res.get('registrantEmail', ''),
+            'Name': res.get('registrantName', ''),
+            'Phone': res.get('registrantTelephone', ''),
+        }
+        creation_date = res.get('created')  # type: ignore
+        registrar = {'Name': res.get('registrarName', '')}
+        domain_status = res.get('status', [])[0] if res.get('status') else None  # or [-1] ?
```

Comment:
?

Reply:
It's not the updated file probably, the line changed to:
domain_status = whois.get('status', [])